src/projects/TodoList/components/CreateNewCategory/index.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import './style.css';
+
+class CreateNewCategory extends React.Component {
+  render () {
+    // logic goes here
+
+    let rgb = [];
+    rgb.push(this.props.newCategoryRed, this.props.newCategoryGreen, this.props.newCategoryBlue)
+    let newCategoryColour = {
+      backgroundColor: `rgb(${rgb})`
+    };
+
+    let cancelCreateCategoryButton = this.props.isNewCategoryBeingCreated ? (
+      <button
+        className="btn btn-outline-danger"
+        onClick={this.props.newCategoryBeingCreatedFalse}
+      >
+        Cancel
+      </button>
+    ) : (
+      null
+    )
```

Comment:
Remember if you've got a ternary like this where one of the results is null you can do it with the `&&` operator instead of the ternary one.

```js
const button = this.props.showButton ? (<button />) : null;
```
can be written as
```js
const button = this.props.showButton && (<button />)
```

Reply:
ah yeah, i'll change that up

Reply:
done